TWLight/graphs/views.py

```diff
@@ -254,13 +254,25 @@ def _write_data(self, response):
             for each_proxy_auth, each_renewed_auth in itertools.zip_longest(
                 proxy_auth_data, renewed_auth_data
             ):
-                each_proxy_auth.extend(
-                    [
-                        each_renewed_auth[1],
-                        str(round(each_renewed_auth[1] / each_proxy_auth[1] * 100, 2))
-                        + "%",
-                    ]
-                )
+                if len(each_renewed_auth) == 2:
```

Comment:
Could you add a comment to explain this guard clause?

Reply:
Added a comment on this.